packages/plugin-types/API.md

```diff
@@ -138,6 +138,12 @@ export type MainStackNameResolver = {
 // @public (undocumented)
 export type ProjectName = string;
 
+// @public (undocumented)
+export type ResolvePathResponse = {
+    branchSecretPath: string;
+    sharedSecretPath: string;
+};
+

```

Comment:
Should we call this `ResolvePathResult` ?
This doesn't involve remote call right result right?

Some "(maybe?) patterns" that exists:

https://github.com/search?q=repo%3Aaws-amplify%2Famplify-backend%20%22Result%20%3D%20%7B%22&type=code

https://github.com/search?q=repo%3Aaws-amplify%2Famplify-backend+%22Response+%3D+%7B%22&type=code

Reply:
You're right it doesn't, just goes to `ParameterPathConversions` for creating the secret paths from `backendIdentifier` so `ResolvePathResult` seems more proper here.